Source/Particles/MultiParticleContainer.H

```diff
@@ -215,6 +216,29 @@ public:
 
     IonizationProcess ionization_process;
 
+    std::string m_B_ext_particle_s = "default";
+    std::string m_E_ext_particle_s = "default";
+    // External fields added to particle fields.
+    amrex::Vector<amrex::Real> m_B_external_particle;
+    amrex::Vector<amrex::Real> m_E_external_particle;
+    // Parser for B_external on the particle
+    std::string m_str_Bx_ext_particle_function;
```

Comment:
Do these really need to be a member of `MultiParticleContainer`? Couldn't they just be local variables?
I am referring to `m_str_Bx_ext_particle_function` for Bx By Bz Ex Ey Ez

Reply:
They need not be members of MultiParticleContainer class. I have moved their definition to the function where the parser is initialized. Thank you!